### PR TITLE
XOR-638 [Fix] Ensure when user try to click the label of the Checkbox , its not affecting the previous checkbox option

### PR DESCRIPTION
**Pull request by NikitaJadhavXoriant**
### Product areas affected

Widgets -> Form  -> Checkbox / Radio field 

### What does this PR do?

Implemented fix so that in Form component, when user try to click the label of the Checkbox, it's not affecting the previous checkbox option

### JIRA ticket

[JIRA](https://weboo.atlassian.net/browse/XOR-638)

### Result

https://user-images.githubusercontent.com/108272606/228465680-31895af8-0a0c-49af-96f4-9ed038e0bce6.mp4

### Checklist

none

### Testing instructions

none

### Deployment instructions

none

### Author concerns

none

### CODE REVIEW

**Review by yogeshmsharma-xoriant — DISMISSED** [review]
looks ok

File: js/libs/core.js [hunk 1]
```diff
@@ -206,6 +206,10 @@ Fliplet.FormBuilder = (function() {
         return this.showLabel || this.showLabel === undefined;
       };
 
+      component.computed._isPreventClicked = function() {
```
Comment (tonytlwu) [inline js/libs/core.js:209]:
```suggestion
      component.computed._labelName = function() {
```

File: js/libs/core.js [hunk 2]
```diff
@@ -206,6 +206,10 @@ Fliplet.FormBuilder = (function() {
...
+        return component.props._componentName.default === 'flCheckbox' || component.props._componentName.default === 'flRadio' ? '' : this.name;
```
Comment (tonytlwu) [inline js/libs/core.js:210]:
```suggestion
        return ['flRadio', 'flCheckbox'].indexOf(component.props._componentName.default) > -1 ? '' : this.name;
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/libs/core.js (hunk 1 above)
Comment (NikitaJadhavXoriant) [reply js/libs/core.js]: done changes

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**